catroid/gradle/standalone_apk_tasks.gradle

```diff
@@ -231,3 +231,18 @@ tasks.whenTaskAdded { task ->
     }
 }
 
+ext.getAdMobAppID = { ->
+    if (project.hasProperty('adMobBannerAppId')) {
+        return project['adMobBannerAppId']
+    } else {
+        return "ca-app-pub-5619801003513533~9206664779"
```

Comment:
Duplicate numbers see comments above?

Reply:
nope, these are values for building with Gradle, which should be removed if the property isn't specified by the user.